lib/actions/deploy.coffee

```diff
@@ -0,0 +1,168 @@
+getBuilderPushEndpoint = (baseUrl, owner, app) ->
+	"https://builder.#{baseUrl}/v1/push?owner=#{owner}&app=#{app}"
```

Comment:
I'd do `encodeURIComponent()` here for the owner and the app name, more future-proof